week1/Test.html

```diff
@@ -0,0 +1,46 @@
+<html>
+<head>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+</head>
+<body>
+  Email <input id = 'email' type = 'text'/><br>
+  Password <input id = 'password' type = 'password'/><br>
+  <button id='login'>Sign Up</button>
+
+  <script>
+  var i = 0;
+  var all = {};
+  var email = $('#email').val();
+  var password = $('#password').val();
+  function storeData(){
```

Comment:
If you are storing data, you shouldn't also be checking the password to see if it's the right URL. I would do one thing one a time. You're doing too much.

Clicking the sign up button should just store the data as an object into an array. That's it.